runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -269,6 +309,8 @@ public XWalkViewInternal(Context context) {
     @XWalkAPI(preWrapperLines = {
                   "        super(${param1}, ${param2});",
                   "        if (isInEditMode()) return;",
+                  "        mAnimatable = ${param2}.getAttributeValue(",
+                  "                XWALK_ATTRS_NAMESPACE, \"animatable\");",
```

Comment:
Actually, I used static member for both of them in previous version, I changed it after the bridge_generator.py can parse the ["]. 
I will modify it as you suggestion.